service/hook/parser/testdata/gitea_push.json

```diff
@@ -17,25 +17,25 @@
     "Sha": "4522cbcefc20728a5b72b3a86af35e608622c514",
     "Message": "Updated readme\n",
     "Author": {
-      "Name": "Unknwon",
```

Comment:
Were all these typos of "Unknwon" intentional?

Reply:
That's not a typo, Unknwon is the name of the author if Gogs ;)

Reply:
I  removed all those "unknwon" changes.